crowbar_framework/config/locales/crowbar/en.yml

```diff
@@ -469,6 +470,17 @@ en:
         raid10: 'RAID 10'
       raid_disks_selected: 'Failed to save %{node}: Not enough disks selected for this RAID level'
 
+  upgrades:
+    show:
+      next: 'Next'
+      title: 'Upgrade to Cloud 6'
+      upgrade_hint: 'Please make sure you have all the latest maintenance updates installed. To start the upgrade process, click Next.'
+    update:
+      title: 'Upgrade to Cloud 6'
+      info: 'Nodes were prepared for upgrade. Click Back to revert this action. Click Next to continue with the Upgrade'
+      reset: 'Reset Upgrade'
+      next: 'Next'
```

Comment:
It would be nice to avoid all these duplicated translation. BTW, for stuff like "Next" we already have global translations.